tests/expanderingsblock-aria.robot

```diff
@@ -0,0 +1,19 @@
+*** Settings ***
+Documentation
+Resource  Resources/keywords_general.robot
+Resource  Resources/keywords_expanderingsblock-aria-expanded.robot
+Library  SeleniumLibrary
+Test Setup  Open Page
+Test Teardown  End Test
+
+*** Variables ***
+${BROWSER} =  chrome
+${URL} =  https://www.vgregion.se/webbhandboken/block/expanderingsblock/
+
+
+
+*** Test Cases ***
+
+Test Expanderingsblock-aria
+
+    Expanderingsblock-aria
```

Comment:
Om du ändå är inne och fixar så kanske du kan lägga in en blankrad i slutet på filerna. gör ingen större skillnad, men är en vanlig konvention.